### PR TITLE
add git remote task

### PR DESCRIPTION
**Pull request by RobinHerbots**
Hi @rubenv (again),

I added the git remote task, I needed it for a project, so I took the effort to implement it in grunt-git.
https://git-scm.com/docs/git-remote.html

Also hoping, to see a quick release so I can start using it right away.

If you have any remarks don't hesitate.

Kind regards,
Robin

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1472,6 +1472,334 @@ grunt.initConfig({
 
 For full details on all the possible codes, please see the [git describe documentation](https://git-scm.com/docs/git-describe).
 
+## The "gitremote" task
+
+The command finds the most recent tag that is reachable from a commit. If the tag points to the commit, then only the tag is shown. Otherwise, it suffixes the tag name with the number of additional commits on top of the tagged object and the abbreviated object name of the most recent commit. The result is a "human-readable" object name which can also be used to identify the commit to other git commands.
```
Comment (rubenv) [inline README.md]: This might be wrong?

**Comment by RobinHerbots** [comment]
@rubenv , 

Oeps, my bad.  

The readme is updated!

**Comment by rubenv** [comment]
Looking good, merging!

**Comment by rubenv** [comment]
Released as `v1.0.12`. Thanks!

**Pull request merged**